src/schemes/boundary/dummy_particles/dummy_particles.jl

```diff
@@ -378,83 +417,128 @@ function compute_pressure!(boundary_model, ::Union{PressureMirroring, PressureZe
+                                          pressure_offset)
     density_neighbor = particle_density(v_neighbor_system, neighbor_system, neighbor)
 
-    resulting_acc = neighbor_system.acceleration -
-                    current_acceleration(system, particle)
+    resulting_acceleration = neighbor_system.acceleration -
+                             current_acceleration(system, particle)
 
     kernel_weight = smoothing_kernel(boundary_model, distance)
 
-    pressure[particle] += (particle_pressure(v_neighbor_system, neighbor_system,
-                                             neighbor) +
-                           dot(resulting_acc, density_neighbor * pos_diff)) *
-                          kernel_weight
+    pressure[particle] += (pressure_offset
+                           +
+                           particle_pressure(v_neighbor_system, neighbor_system,
+                                             neighbor)
+                           +
+                           dynamic_pressure(boundary_density_calculator, density_neighbor,
+                                            v, v_neighbor_system, pos_diff, distance,
+                                            particle, neighbor, system, neighbor_system)
+                           +
+                           dot(resulting_acceleration,
+                               density_neighbor * pos_diff)) * kernel_weight
 
     cache.volume[particle] += kernel_weight
 
     compute_smoothed_velocity!(cache, viscosity, neighbor_system, v_neighbor_system,
                                kernel_weight, particle, neighbor)
 end
 
+@inline function dynamic_pressure(boundary_density_calculator, density_neighbor, v,
+                                  v_neighbor_system, pos_diff, distance, particle, neighbor,
+                                  system,
+                                  neighbor_system)
+    return zero(density_neighbor)
+end
+
+@inline function dynamic_pressure(boundary_density_calculator::BernoulliPressureExtrapolation,
+                                  density_neighbor, v, v_neighbor_system, pos_diff,
+                                  distance, particle, neighbor, system::BoundarySystem,
+                                  neighbor_system)
+    if system.ismoving[]
+        relative_velocity = current_velocity(v, system, particle) .-
+                            current_velocity(v_neighbor_system, neighbor_system, neighbor)
+        normal_velocity = dot(relative_velocity, pos_diff) / distance
+        return boundary_density_calculator.factor * density_neighbor *
+               dot(normal_velocity, normal_velocity) / 2
+    end
+    return zero(density_neighbor)
+end
+
+@inline function dynamic_pressure(boundary_density_calculator::BernoulliPressureExtrapolation,
+                                  density_neighbor, v, v_neighbor_system, pos_diff,
+                                  distance, particle, neighbor, system::SolidSystem,
+                                  neighbor_system)
```

Comment:
```suggestion
                                  distance, particle, neighbor, 
                                  system::SolidSystem, neighbor_system)
```

Reply:
?

Reply:
It's my personal style. See comment above